pyspecProcScripts/phasing.py

```diff
@@ -320,6 +321,8 @@ def filter_ranges(B, A):
     peakrange = peakrange[0]
     frq_center = np.mean(peakrange).item()
     frq_half = np.diff(peakrange).item() / 2
+    d.set_prop(peakrange, frq_half)
+    print(peakrange)
```

Comment:
This is fine for debugging, but don't include it in the final version that you want to merge